plugin/slimux.vim

```diff
@@ -215,8 +215,16 @@ function! s:Send(tmux_packet)
       endif
 
       let named_buffer = s:tmux_version >= '2.0' ? '-b Slimux' : ''
+      if exists('g:slimux_python_ipython')
```

Comment:
I actually think this option is `python` related, and should not be included in `slimux.vim`.
For example, I think to use hooks is better: https://github.com/epeli/slimux/issues/38#issuecomment-183373645